packages/python/BUILD

```diff
@@ -0,0 +1,89 @@
+# Copyright (c) 2015 Nuxi, https://nuxi.nl/
+#
+# This file is distributed under a 2-clause BSD license.
+# See the LICENSE file for details.
+
+import os
+import sysconfig
+
+def build(ctx):
+    srcdir = ctx.extract('Python-3.6.0a2')
+
+    # Delete the in-tree copy of these, to make certain the out-of-tree
+    # versions are used, i.e. those generated from our patched importlib
+    os.unlink(os.path.join(str(srcdir), 'Python', 'importlib.h'))
+    os.unlink(os.path.join(str(srcdir), 'Python', 'importlib_external.h'))
+
+    # First build a native binary, this will be used by the cross-build
+    # to generate some files (see also patch-makefile-regen)
+    hostbuild = srcdir.host().gnu_configure()
+    hostbuild.make()
+
+    build = srcdir.gnu_configure([
+        '--build=%s' % sysconfig.get_config_var('HOST_GNU_TYPE'),
```

Comment:
This line is present because CPython requires `--build=...` to complete a cross-build. On the basis of "explicit it better than implicit' I prefer explicitly building the string to a keyword argument. It might be an idea for the `ctx` object to have e.g. `host_arch`, `build_arch` properties or methods. Rather than import `sysconfig`.